### ISSUE TITLE
Reproduding QM9 Results

### ISSUE THREAD

**Issue by YZHANG1996** [torchmd/torchmd-net#123]
I am training ET on QM9, trying to reproduce the results reported in the paper.
I used the parameters in ET-QM9.yaml without tuning, and the performances for HOMO and R2 are as follows.
<img width="541" alt="image" src="https://user-images.githubusercontent.com/101861396/188294867-fe7026f9-ccfc-41d5-9465-453dbfcdca24.png">
<img width="531" alt="image" src="https://user-images.githubusercontent.com/101861396/188294869-866cb0cb-f53f-4eca-8077-583d63aed779.png">
Is there some recommendation for parameters selection?

**Comment by YZHANG1996** [comment]
Another inquiry is that, I found validation loss far smaller than test loss (e.g. epoch=659-val_loss=0.0013-test_loss=0.0232).
But since overfitting should only happen on training set, the validation loss should be on par with the test loss. Is there something wrong with my understanding?

**Comment by giadefa** [comment]
Validation loss is MSE, test loss is MAE.

**Comment by PhilippThoelke** [comment]
Your test score for HOMO (23.2 meV) is close to what we report in the paper (20.3 meV), the difference likely stems from different random seeds.

We used a specialized output network to predict R2 (see equation 13 in the paper). You can select the output network by setting `output_model` to `ElectronicSpatialExtent` in the config yaml file.
